lib/schema/translation_tooling.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+
+fact_schema = YAML.load_file(File.join(File.dirname(__FILE__), "facter.yaml"))
+facts = []
+
+fact_schema.each do |fact|
+  fact_name = fact[0]
+  fact_details = fact[1]
+
+  fact_data = {:name => fact_name,
+               :description => fact_details['description'],
+               :resolution => fact_details['resolution'],
+               :caveats => fact_details['caveats']}
+
+  facts << fact_data
+end
+
+File.open(File.join(File.dirname(__FILE__), "core_facts.pot"), 'w') do |file|
+  file.puts <<-HEADER
+# CORE FACTS SCHEMA
+# Copyright (C) 2016 Puppet Labs, LLC
+# This file is distributed under the same license as the Sinatra i18n demo package.
+# FIRST AUTHOR <info@puppet.com>, 2016.
+#
+#, fuzzy
+# msgid ""
+# msgstr ""
+"Project-Id-Version: Sinatra i18n demo init-11-ga552a06\\n"
+"\\n"
+"Report-Msgid-Bugs-To: docs@puppetlabs.com\\n"
+"POT-Creation-Date: 2016-08-26 09:00-0000\\n"
+"PO-Revision-Date: #{Time.now}\\n"
```

Comment:
In the Leatherman `.pot` file this is a pattern: `"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"`. I don't think we want the actual date. I'd use the leatherman.pot file as a guide for what the header should look like.